.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
 
   plugins: [
     'jest',
+    'no-for-of-loops',
```

Comment:
Do you need a whole plugin for this? You can use no-restricted-syntax to block for..of loops with a custom message.